lightningd/channel_gossip.c

```diff
@@ -66,71 +187,176 @@ struct channel_gossip {
 
 	/* Details of latest channel_update sent by peer */
 	const struct peer_update *peer_update;
+
+	/* To avoid a storm, we only respond to announcement_signatures with
+	 * our own signatures once */
+	bool sent_sigs;
```

Comment:
It's nice to have the announcement signatures loop fixed!  It made for some long CI logs.